l10n_br_sale_stock/models/stock.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+# (C) 2016 Luis Felipe Mileo - KMEE
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from openerp import api, models
+
+
+class StockMove(models.Model):
+    _inherit = 'stock.move'
+
+    @api.multi
+    def action_confirm(self):
+        """
+            Pass the department_id and the dispatch_note to the picking)
+        """
```

Comment:
looks like the docstring doesn't match what the code does, better remove it then.